tests/e2e/dynamic_provisioning_test.go

```diff
@@ -158,6 +158,7 @@ var _ = Describe("[fsx-csi-e2e] Dynamic Provisioning with s3 data repository", f
 						Parameters: map[string]string{
 							"subnetId":         subnetId,
 							"securityGroupIds": strings.Join(securityGroupIds, ","),
+							"autoImportPolicy": "NEW_CHANGED"
```

Comment:
think a comma is missing here.

Also might need to run gofmt, the "verify" test complained about it